recipes-security/optee/optee-os/0001-imx7ulp-reserve-a-watchdog-for-OP-TEE.patch

```diff
@@ -0,0 +1,42 @@
+From 7a8985291df01d5e66fa2383c4086e7ab86c7724 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Tue, 10 Dec 2019 21:52:50 +0100
+Subject: [PATCH] imx7ulp: reserve a watchdog for OP-TEE
+
+This watchdog will be used via PSCI for system reset.
+
+Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>
+---
+ core/arch/arm/plat-imx/main.c | 16 ++++++++++++++++
+ 1 file changed, 16 insertions(+)
+
+diff --git a/core/arch/arm/plat-imx/main.c b/core/arch/arm/plat-imx/main.c
+index 602365b5..7b52f01a 100644
+--- a/core/arch/arm/plat-imx/main.c
++++ b/core/arch/arm/plat-imx/main.c
+@@ -140,6 +140,22 @@ void console_init(void)
+ #endif
+ }
+ 
++/* reserve the following drivers for exclusive OP-TEE access */
++ const char *main_get_optee_exclusive_node_name(unsigned int i)
++{
++	static const char *exclusive_drivers[] = {
++#if defined CFG_MX7ULP
```

Comment:
Care to add a comment here explaining why this driver needs to be handled by optee only?